packages/plugins/rum/src/validate.ts

```diff
@@ -56,6 +59,9 @@ export const validateSDKOptions = (options: Options): ToReturn<SDKOptionsWithDef
     const toReturn: ToReturn<SDKOptionsWithDefaults> = {
         errors: [],
     };
+    if (!validatedOptions.sdk || validatedOptions.sdk.disabled) {
+        return toReturn;
+    }
```

Comment:
With this `if` as an early out, I think you can remove the one right below, `if (validatedOptions.sdk)`.